### ISSUE TITLE
GMPE object has no attribute 'DEFINED_FOR_REFERENCE_VELOCITY

### ISSUE THREAD

**Issue by elisa82** [gem/oq-engine#5196]
I have a problem running an hazard analysis in terms of AvgSA as intensity measure. 

If I use the GMPE AkkarEtAlRepi2014 the code gives this error:
```
ValueError: 'AkkarEtAlRepi2014' object has no attribute 'DEFINED_FOR_REFERENCE_VELOCITY' in file...
```
`generic_gmpe_avgsa.py` seems to be not complete since I found these comments inside the file:

 ``` 
# Check if this GMPE has the necessary requirements
# TO-DO
```
Please can you check?
thanks

**Comment by daniviga** [comment]
Dear @elisa82,

thanks for the feedback provided. However for the support requests please use the support forum: https://groups.google.com/forum/#!forum/openquake-users

**Comment by daniviga** [comment]
@elisa82 bug has been fixed and it's already available in `master`. It will we be also available in the next maintenance release (3.7.1)

**Comment by elisa82** [comment]
Thank you very much!
